README.md

```diff
@@ -1,2 +1,16 @@
 # veryrezsi
 Subscription and expense calculator web-application.
+
+# Usage 
+
+## Prerequisites 
+
+- npm v7 or higher
+  - 8.4.1 ✅
+- node v14 or higher
+  - v16.14.0 ✅
+
+## Development
+- `cd ./veryrezsi/client`
+- `npm i`
+- `npm run dev`
```

Comment:
good stuff, I was planning on writing a readme, thanks, I will write the server counterpart then

Reply:
Yep, it was just the easy one.